dbt/adapters/athena/impl.py

```diff
@@ -95,14 +95,14 @@ class AthenaConfig(AdapterConfig):
     partitioned_by: Optional[str] = None
     bucketed_by: Optional[str] = None
     bucket_count: Optional[str] = None
-    table_type: Optional[str] = "hive"
-    ha: Optional[bool] = False
-    format: Optional[str] = "parquet"
+    table_type: str = "hive"
+    ha: bool = False
```

Comment:
isn't ha Optional?

Reply:
Since there is a default we don't need it I think

Reply:
But then let's follow the same for other parameters - I saw that for some we have Optional still. Let's be consistent and keep one approach.

Reply:
Hum this will be tricky because sometimes we assume this has a default value (i.e. we assume that None value is not possible, therefore having the Optional typing is not right)